### DISCUSSION TITLE
How to use extracted Principal Components as a Data Source for Supervised Learning?

### DISCUSSION THREAD

**Discussion by Hertzspr** [teamalgoritma/community#137] [Q&A] [answered]
Bagaimana cara menggunakan hasil ekstrak PC dari PCA sebagai basis data untuk supervised learning? 
![Capture](https://user-images.githubusercontent.com/95233522/174248447-f5ef49da-955f-41f9-92b2-d22ce4490828.JPG)

Hasil PC nya kan nggak ada labelnya? Atau saya ada gagal paham mengenai data PC?
![label](https://user-images.githubusercontent.com/95233522/174248470-5df303ae-5fec-4a0e-be98-c67a993f6597.JPG)

**Comment by Hertzspr** [comment, marked as answer]
Udah ketemu untuk regression -> 
[contoh Principal Component Regression](https://rpubs.com/esobolewska/pcr-step-by-step)

Ternyata bisa diginiin. Oops. Maaf waktu nanya saya lupa, baru sadar kalo dari awal preprocessing label targetnya bisa dipisah, kalo nggak ada ketika waktu mau supervised learning, ya tinggal dipasang lagi yang sebelumnya udah kepisah. 

https://rpubs.com/esobolewska/pcr-step-by-step

**Reply by Hertzspr** [reply]
Cuma jadi ga interpretable ya karena predictornya berupa PC. Duh, serba salah.

**Comment by hwulanayu** [comment]
Dear @Hertzspr,

untuk contoh penggunaan PCA pada kasus klasifikasi, berikut adalah beberapa referensi penggunaan PCA pada kasus klasifikasi dengan menggunakan library `recipes`: 

- https://rpubs.com/tomytjandra/PCA-before-classification. 
- https://algotech.netlify.app/blog/time-and-accuracy-improvement-using-pca/ (pada bagian `APPLYING PCA IN BREAST CANCER DATASET`

Benar bahwa ketika kita menggunakan nilai PCA sebagai prediktor, maka model tersebut akan sulit diinterpretasi karena nilai tiap PCA merupakan nilai gabungan/campuran dari beberapa variabel prediktornya.
